packages/blade/src/components/Button/BaseButton/StyledBaseButton.native.tsx

```diff
@@ -18,13 +18,26 @@ const StyledPressable = styled(Animated.createAnimatedComponent(Pressable))<
   return {
     ...getStyledBaseButtonStyles(props),
     alignSelf: 'center',
+    display: 'flex',
     ...styledPropsCSSObject,
   };
 });
 
+const openURL = async (href: string): Promise<void> => {
```

Comment:
can't we use the link's openURL directly here?

Reply:
Didnt want to import Link component inside Button but I think we can move this to utils and use at both places. Didn't add it then because its a small function that just calls React Native's Linking.openURL and throws error and error message is different in both.